kibana-details/impersonation.md

```diff
@@ -0,0 +1,137 @@
+# Impersonation 
+
+After describing what [the impersonation is](../kibana.md#impersonate), it's high time to see how does ROR support it and who and when could be interested in using this feature. Let's start with the latter one.
+
+## Use cases
+
+The impersonation feature is prepared for a ROR admin. It's not rather useful for users (at least not directly). We can point out two the most obvious use cases when the admin could take advantage of the feature:
+
+#### Debugging users' problems:
+
+Let's imagine that some user has a problem with its ROR configuration (eg. the user doesn't has an access to some feature that was blocked at ROR's level by you, the admin). They are not able to clearly describe what the issue is (sounds familiar?). For the admin, it'd be better if they can see what the user sees. Thanks to impersonation feature, an admin is allowed to impersonate the user and feel and see the same the user sees. 
```

Comment:
the user doesn't have access

.., and they are not able to clearly describe

As an administrator, it would be extremely beneficial  if you could see what he user sees.